CONTRIBUTING.md

```diff
@@ -87,11 +87,29 @@ connected Android device, from that directory you would run:
 
  * `flutter run --engine-src-path /path/to/engine/src -t tabs.dart`
 
+You can replace `--engine-src-path /path/to/engine/src` with just
+`--engine-debug` if you place the Flutter engine repository in the
+`.../engine/src` directory and the Flutter framework ("flutter")
+repository in the `.../flutter` directory, so that `engine` and
+`flutter` are peers.
+
 If you're going to be debugging crashes in the engine, make sure you add
 `android:debuggable="true"` to the `<application>` element in the
 `android/AndroidManifest.xml` file for the Flutter app you are using
 to test the engine.
 
+To build a release engine, run the following steps, again from the
+`src` directory created in the steps above:
+
+ * `gclient sync` to update your dependencies.
+ * `./sky/tools/gn --android --release` to prepare your build files to build an Android Release binary.
```

Comment:
I think @jason-simmons removed the `--release` flag from this script.  Release is now just the absence of `--debug`.  Maybe we should add `--release` as an alias for user friendliness?